vtysh/vtysh_config.c

```diff
@@ -131,6 +131,17 @@ static void config_add_line_uniq(struct list *config, const char *line)
 	listnode_add_sort(config, XSTRDUP(MTYPE_VTYSH_CONFIG_LINE, line));
 }
 
+static void config_add_line_end(struct list *config, const char *line)
+{
+	struct listnode *node;
+	void *item = XSTRDUP(MTYPE_VTYSH_CONFIG_LINE, line);
+
+	listnode_add(config, item);
+	node = listnode_lookup(config, item);
+	if (node)
+		listnode_move_to_tail(config, node);
```

Comment:
doesn't `listnode_add()` already add the element to the end of the list?

Reply:
It does, I was concerned about the explicitness of what is going on here enough that I thought this warranted it's own special code.  I can remove and just use listnode_add if you want?

Reply:
I think it's ok then, given that this isn't performance critical code.